client/js/analysis.js

```diff
@@ -1,15 +1,123 @@
-  const el = document.getElementById(elId); 
-  el.innerText = value;
-  return value;
+window.onload = function() {
+  for (let i = 1; i < 6; i++) {
+    setTimeout(() => {
+      addListElement(i);
+    }, i * 1500);
+  }
+  setTimeout(() => {
+    addFeedbackResult('POSITIVE');
+  }, 9000);
+
+  for (let i = 0; i < 6; i++) {
+    const values = [
+      'TOXICITY',
+      'IDENTITY ATTACK',
+      'INSULT',
+      'PROFANITY',
+      'THREAT',
+      'FLIRTATION',
+    ];
+    addDonutChart(values[i], Math.floor(Math.random() * 61) + 30);
+  }
+};
+
+/**
+ * Adds an bullet point to the comment list
+ *
+ * @param {string} str the txt of the bullet point
+ */
+function addListElement(str) {
+  const el = document.createElement('li');
+  el.innerHTML = str;
+  document.getElementById('list').appendChild(el);
+}
+
+/**
+ * Adds the feedback result element to the DOM tree
+ *
+ * @param {string} result Positive, Negative, or Neutral
+ */
+function addFeedbackResult(result) {
+  const el = document.createElement('div');
+  el.classList = 'center fade';
+  const text = document.createElement('h1');
+  text.setAttribute('id', 'overall');
+  text.innerHTML = 'Overall Response\n: ' + result;
+  el.appendChild(text);
+
+  const list = document.getElementById('list');
+  list.appendChild(el);
+}
+
+/**
+ * Adds a donut chart
+ *
+ * @param {string} str the header string of the circle
+ * @param {string} percent the amount the circle will be filled
+ */
+function addDonutChart(str, percent) {
+  const div = document.createElement('div');
+
+  div.setAttribute('id', 'a-chart');
+  div.className = 'item donut';
+  div.style.setProperty('--percent', percent);
+
+  const header = document.createElement('h2');
+  header.innerText = str;
+
+  div.appendChild(header);
+
+  const svg = createSVGElement('svg');
+
+  svg.setAttribute('xmlns', 'http://www.w3.org/2000/svg');
+
+  const g = buildCircle();
+  svg.appendChild(g);
+
+  div.appendChild(svg);
+
+  const currentDiv = document.getElementById('charts');
+  currentDiv.appendChild(div);
+}
+
+/**
+ * builds the circle parts of the donut charts
+ */
+function buildCircle() {
+  const g = createSVGElement('g');
+
+  const insideCircle = createSVGElement('circle');
+
+  insideCircle.setAttribute('r', '70');
+  insideCircle.setAttribute('cx', '50%');
+  insideCircle.setAttribute('cy', '50%');
+  insideCircle.setAttribute('stroke-width', '25');
+  insideCircle.setAttribute('stroke', '#d3d3d3');
+  insideCircle.setAttribute('fill', 'none');
+
+  const outsideCircle = createSVGElement('circle');
+
+  outsideCircle.setAttribute('class', 'circle_animation');
+  outsideCircle.setAttribute('r', '70');
+  outsideCircle.setAttribute('cx', '50%');
+  outsideCircle.setAttribute('cy', '50%');
+  outsideCircle.setAttribute('stroke-width', '25');
+  outsideCircle.setAttribute('stroke', '#ff1493'); // ff073a is good too
+  outsideCircle.setAttribute('fill', 'none');
+  outsideCircle.setAttribute('pathLength', '100');
```

Comment:
might be able to avoid setting so many attributes at once by adding properties in css file like how ibiyemi did here for `verticalLine` svg:

- [css lines 46-51](https://github.com/googleinterns/step99-2020/pull/42/files#diff-df3e422d362ce99ee9375a71adbddf44R46-R51)
- [js lines 260-266](https://github.com/googleinterns/step99-2020/pull/42/files#diff-349c8b5f8ae66be1a9cfbe2f7817691dR260-R266)

Reply:
The only things I could bring to css would be the stroke-width and fill, but r, cx, cy, pathLength, and stroke all need to be in js

Reply:
I would prefer to keep it this way

Reply:
sounds good